tools/src/launcher/Main.hs

```diff
@@ -491,3 +546,19 @@ ignoreSIGPIPE = handle (\ex -> case ex of
         , ioe_errno = Just ioe }
         | Errno ioe == ePIPE -> return ()
     _ -> throwIO ex )
+
+----------------------------------------------------------------------------
+-- SIGKILL
+----------------------------------------------------------------------------
+
+-- | If we're on Linux or macOS, send a SIGKILL to a process.
+maybeTrySIGKILL :: ProcessHandle -> M ()
+maybeTrySIGKILL _h = do
+#ifdef mingw32_HOST_OS
+    logInfo "Not trying to send a SIGKILL because we're on Windows"
```

Comment:
Is there really no way to ask a process to terminate on Windows?

It's not very good to have behaviors diverge between OSes, in the long run..

Reply:
According to MSDN, `terminateProcess` is already as powerful as it gets

Reply:
The problem on Windows is that there is no standard equivalent of SIGTERM, of asking nicely for a process to terminate. There is only `TerminateProcess()` which is forcible and gives no chance to clean up.

http://www.drdobbs.com/a-safer-alternative-to-terminateprocess/184416547